pallets/liquidity-provider/src/lib.rs

```diff
@@ -56,26 +57,30 @@ decl_module! {
         #[weight = 0]
         pub fn attest(
             origin,
-            collateral: Collateral,
+            asset: Asset,
             balance: Balance<T>
         ) -> dispatch::DispatchResult
         {
             let who = ensure_signed(origin)?;
+            do_attest::<T>(who, asset, balance)
+        }
 
+        #[weight = 0]
+        pub fn mint(
```

Comment:
Still not sure about exposing this as a public dispatchable call, maybe as a private function of the module?

Reply:
mint method was removed alongside with mint members